spec/helper/revenuecat_internal_helper_spec.rb

```diff
@@ -408,4 +408,25 @@ def setup_stubs
       )
     end
   end
+
+  describe '.replace_in' do
```

Comment:
Now that we have tests for this, could you add one that would cover what I fixed in https://github.com/RevenueCat/purchases-ios/pull/1536?
Example:
- Input: `55413C0025B778E200ECCA5A` 
- Replace: `4.1.3`
- Replacement: `4.4.0-SNAPSHOT`

I see that you copied over the correct implementation with the escaped `.`, so this shouldn't replace anything :)

Reply:
So tests don't run fastlane `sh` actions by default so I couldn't actually test the command itself. But I thought there was value in actually testing the command in this case (pretty simple command). So I made `sh` actions run only for these tests by setting the environment variables `ENV["FORCE_SH_DURING_TESTS"]`. I wouldn't like that for production code but I think it's ok for tests, but lmk what you think!

Reply:
I think `FORCE_SH_DURING_TESTS` for this subgroup of tests makes sense.